tests/Unit/ValueObject/SearchUrlBuilderTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\LuigisBoxBundle\Tests\Unit\ValueObject;
+
+use Answear\LuigisBoxBundle\ValueObject\Search\Context;
+use Answear\LuigisBoxBundle\ValueObject\SearchUrlBuilder;
+use PHPUnit\Framework\TestCase;
+
+class SearchUrlBuilderTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function buildValidUrlTest(): void
+    {
+        $searchBuilder = new SearchUrlBuilder('123-345');
+        $this->assertSame('size=10&tracker_id=123-345&page=1', $searchBuilder->toUrlQuery());
```

Comment:
Size and tracker_id are constant?

Reply:
There is no need. IMO this will complicate the reading of test.